### ISSUE TITLE
Cannot click off patch notes

### ISSUE THREAD

**Issue by wmartin1986** [tirem/HXUI#52]
Please just remove this window. Everyone I know struggles to click the patch notes off.

**Comment by cowlinator** [comment]
Yes, please just remove the patch notes window. I struggled for like 10 minutes to close it. I never succeeded. I just sqashed the window down to a tiny square. But because i never closed it, it popped up again when i logged back in?? Why would i want to see it twice?

In fact, i dont want to see it at all. If i want to know the patch notes, i will just go look them up.

**Comment by RetroFrito** [comment]
Yep, I just started using this and the mouse behavior is so odd. The mouse cursor disappears when I go over the HXUI windows. I can interact with some menu items but not the X. I cannot figure out how to drag the UI items where I want them. I have tried setting my resolution the same in each of the fields available in the Horizon launcher, and even tried lowering the res to 720p and it's still not working.

**Comment by bokbokker** [comment]
+1 on this. Impossible to click the little X to remove patch notes...

**Comment by Shuu-37** [comment]
@wmartin1986 @cowlinator @RetroFrito @bokbokker are you all playing in window mode?

**Comment by Shuu-37** [comment]
Should be fixed as of new release.
